misc/build_helpers/build-manylinux-wheels.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+# This runs in the container to actually build the wheels.
+BUILDER="/io/misc/build_helpers/_build-wheels.sh"
+
+# This image can build x86_64 (64 bit) manylinux wheels.
+DOCKER_IMAGE="quay.io/pypa/manylinux1_x86_64"
```

Comment:
hmm a docker image from quay.io? it's too bad there isn't a threshold signature scheme at work here to prevent us to installing a bad container image in the case where quay.io get hacked.